Userland/Applications/PixelPaint/Tools/MoveTool.cpp

```diff
@@ -101,21 +139,22 @@ void MoveTool::on_mouseup(Layer* layer, MouseEvent& event)
         return;
 
     if (m_scaling) {
-        m_editor->active_layer()->resize(m_new_layer_size, m_new_scaled_layer_location, Gfx::Painter::ScalingMode::BilinearBlend);
+        m_editor->active_layer()->resize(m_new_layer_rect, Gfx::Painter::ScalingMode::BilinearBlend);
         m_editor->layers_did_change();
     }
 
     m_scaling = false;
     m_layer_being_moved = nullptr;
     m_cached_preview_bitmap = nullptr;
+    m_new_layer_rect = m_editor->active_layer()->rect();
     m_editor->update_tool_cursor();
     m_editor->did_complete_action(tool_name());
 }
 
 bool MoveTool::on_keydown(GUI::KeyEvent& event)
 {
     if (event.key() == Key_Shift)
-        m_keep_ascept_ratio = true;
+        m_keep_aspect_ratio = true;
```

Comment:
I think this should be in a separate commit

Reply:
split into a separate commit